Cue/app/tabs/library/LibraryHome.js

```diff
@@ -111,7 +111,16 @@ class LibraryHome extends React.Component {
       [
         {text: 'Cancel', style: 'cancel'},
         {text: 'Create', onPress: (deckName) => {
-          if (deckName && deckName.length) {
+          if (!deckName || !deckName.length) {
+            Alert.alert(
+              Platform.OS === 'android' ? 'Enter a deck name' : 'Enter a Deck Name',
+            )
+          } else if (deckName.length > 255) {
+            Alert.alert(
+              Platform.OS === 'android' ? 'Deck name too long' : 'Deck Name Too Long',
+              'Please enter a shorter deck name',
```

Comment:
-----
**Deck Name Too Long**

Use a shorter name for your deck.

-----
? I don't feel too strongly about either, but I'd avoid using the word "please".